sql/core/src/test/scala/org/apache/spark/sql/DataFrameStatSuite.scala

```diff
@@ -150,6 +150,19 @@ class DataFrameStatSuite extends QueryTest with SharedSQLContext {
       assert(math.abs(d1 - 2 * q1 * n) < error_double)
       assert(math.abs(d2 - 2 * q2 * n) < error_double)
     }
+
+    for (epsilon <- epsilons) {
+      val Array(Array(s1, s2), Array(d1, d2)) = df.stat.approxQuantile(Array("singles", "doubles"),
```

Comment:
minor, but we could just have this test added to the above `for (epsilon <- epsilons)` loop.